physionet-django/user/management/commands/reject_pending_credentialing_applications.py

```diff
@@ -0,0 +1,70 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.db import transaction
+from django.http import HttpRequest
+from django.utils import timezone
+
+import notification.utility as notification
+from user.models import CredentialApplication, User
+
+LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NUMBER_OF_APPLICATIONS_TO_REJECT = 5
+
+
+def get_application_to_be_rejected(number_of_applications=DEFAULT_NUMBER_OF_APPLICATIONS_TO_REJECT):
+    """
+    Get all CredentialApplication that has been pending for more than
+    settings.MAX_REFERENCE_VERIFICATION_DAYS_BEFORE_AUTO_REJECTION days.
+    """
+    today = timezone.now()
+    limit = today - timezone.timedelta(days=settings.MAX_REFERENCE_VERIFICATION_DAYS_BEFORE_AUTO_REJECTION)
+
+    # get all applications that have decision pending
+    applications = CredentialApplication.objects.filter(status=0)
+    # get applications with Reference Response pending
+    response_applications = applications.filter(credential_review__status=30)
+    # finally get applications that have been pending for more than the limit
+    filtered_applications = response_applications.filter(reference_contact_datetime__lt=limit)[:number_of_applications]
+    return filtered_applications
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("-n", "--number", type=int, help="Number of applications to be rejected")
+
+    def handle(self, *args, **options):
+        """
+        Delete all Credentialing applications whose reference checks have Awaiting Reference Response pending after
+        waiting for settings.MAX_REFERENCE_VERIFICATION_DAYS_BEFORE_AUTO_REJECTION days.
+        """
+
+        # only perform auto rejection if the setting.ENABLE_CREDENTIALING_AUTO_REJECTION is True
+        if not settings.ENABLE_CREDENTIALING_AUTO_REJECTION:
+            LOGGER.info('Auto rejection of credentialing applications is disabled. Exiting.')
+            LOGGER.warning('If this was unintentional, please set '
+                           'MAX_REFERENCE_VERIFICATION_DAYS_BEFORE_AUTO_REJECTION to True in .env file.')
+            return
+
+        # total number of applications to be rejected
+        total_applications_to_reject = options['number'] or DEFAULT_NUMBER_OF_APPLICATIONS_TO_REJECT
+
+        LOGGER.info(f'Total number of applications to be rejected: {total_applications_to_reject}')
```

Comment:
I find this log message confusing. I suggests to me that X applications are going to be rejected, when actually this is just the maximum possible number of applications that would be rejected. 

We could reword the message, but I think it's probably okay to just remove it. I don't think it is saying anything particularly useful.

Reply:
Thats a good point. I like the removing idea. Let me push changes